pkgs/applications/misc/dbeaver/default.nix

```diff
@@ -18,13 +18,13 @@
 
 stdenv.mkDerivation rec {
   pname = "dbeaver";
-  version = "21.2.3"; # When updating also update fetchedMavenDeps.sha256
+  version = "21.2.4"; # When updating also update fetchedMavenDeps.sha256
```

Comment:
Please see the comment

Reply:
`fetchedMavenDeps.sha256` did not change, just like e.g. in [21.1.5 -> 21.2.0](https://github.com/NixOS/nixpkgs/commit/526d290b7ce8391eafb693af09281ebaa09bf00b) and [21.0.1 -> 21.0.2](https://github.com/NixOS/nixpkgs/commit/835a9598eea863bfd7cf5fd31e7d0b932c23c0bb). 

I'm always setting `fakeSha256` before building.